packages/server/lib/controllers/auth.controller.ts

```diff
@@ -26,6 +28,16 @@ export interface WebUser {
     name: string;
 }
 
+interface InviteAccountState {
+    accountId: number;
+    token: string;
+}
+
+let workos: WorkOS | null = null;
+if (process.env['WORKOS_API_KEY']) {
```

Comment:
I would have the server throw and not start if workOS (or anything else actually) is not configured properly. Instead of checking if workOS is not null at request time